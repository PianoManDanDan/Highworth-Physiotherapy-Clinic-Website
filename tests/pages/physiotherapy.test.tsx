import React from 'react';
import { getPage } from 'next-page-tester';

describe('Physiotherapy Page', () => {
  test('matches the snapshot', async () => {
    const { render } = await getPage({
      route: '/physiotherapy',
    });

    const page = render();

    expect(page).toMatchSnapshot();
  });
});
