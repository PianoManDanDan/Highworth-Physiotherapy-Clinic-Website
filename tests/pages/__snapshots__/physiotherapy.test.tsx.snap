// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Physiotherapy Page matches the snapshot 1`] = `
Object {
  "nextRoot": <div
    id="__next"
  >
    <header>
      <div>
        <a
          href="/"
        >
          <div
            style="display:inline-block;max-width:100%;overflow:hidden;position:relative;box-sizing:border-box;margin:0"
          >
            <div
              style="box-sizing:border-box;display:block;max-width:100%"
            >
              <img
                alt=""
                aria-hidden="true"
                role="presentation"
                src="data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMzAiIGhlaWdodD0iMTI4IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZlcnNpb249IjEuMSIvPg=="
                style="max-width:100%;display:block;margin:0;border:none;padding:0"
              />
            </div>
            <noscript>
              <img
                alt="Highworth Physiotherapy Clinic"
                decoding="async"
                src="/assets/logos/hpc.png"
                srcset="/_next/image?url=%2Fassets%2Flogos%2Fhpc.png&w=32&q=75 1x, /_next/image?url=%2Fassets%2Flogos%2Fhpc.png&w=64&q=75 2x"
                style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
              />
            </noscript>
            <img
              alt="Highworth Physiotherapy Clinic"
              decoding="async"
              src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
              style="position:absolute;top:0;left:0;bottom:0;right:0;box-sizing:border-box;padding:0;border:none;margin:auto;display:block;width:0;height:0;min-width:100%;max-width:100%;min-height:100%;max-height:100%"
            />
          </div>
        </a>
        <div>
          <a
            href="/"
          >
            <h1>
              Highworth Physiotherapy Clinic
            </h1>
          </a>
          <address>
            <h2>
              <a
                href="tel:+441793763814"
              >
                <svg
                  fill="none"
                  height="24"
                  stroke="currentColor"
                  stroke-linecap="round"
                  stroke-linejoin="round"
                  stroke-width="2"
                  viewBox="0 0 24 24"
                  width="24"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M22 16.92v3a2 2 0 0 1-2.18 2 19.79 19.79 0 0 1-8.63-3.07 19.5 19.5 0 0 1-6-6 19.79 19.79 0 0 1-3.07-8.67A2 2 0 0 1 4.11 2h3a2 2 0 0 1 2 1.72 12.84 12.84 0 0 0 .7 2.81 2 2 0 0 1-.45 2.11L8.09 9.91a16 16 0 0 0 6 6l1.27-1.27a2 2 0 0 1 2.11-.45 12.84 12.84 0 0 0 2.81.7A2 2 0 0 1 22 16.92z"
                  />
                </svg>
                <span>
                  01793 763814
                </span>
              </a>
            </h2>
          </address>
        </div>
      </div>
      <nav>
        <a
          href="/"
        >
          Home
        </a>
        <a
          href="/facilities"
        >
          Facilities
        </a>
        <a
          href="/physiotherapy"
        >
          Physiotherapy
        </a>
        <a
          href="/animals"
        >
          Animals
        </a>
        <a
          href="/meet-the-team"
        >
          Meet the Team
        </a>
        <a
          href="/faq"
        >
          FAQ's
        </a>
        <a
          href="/contact"
        >
          Contact Us
        </a>
      </nav>
    </header>
    <main>
      <h1>
        PHYSIOTHERAPY
      </h1>
      <p>
        Physiotherapy is the treatment of a wide variety of injuries and conditions using a range of manual techniques, manipulation, massage, advice, exercise and electrotherapy. When appropriate, a lot of time is spent teaching the patient self help exercises so that they can manage their own condition. All the physiotherapists at this practice are experienced Chartered Physiotherapists who treat sports injuries, back pain, neck pain, arthritis, aches, strains and pains. We are committed to a high level of clinical expertise and customer satisfaction.
      </p>
      <p>
        All our physiotherapists undertake at least 25 hours post graduate education every year.
      </p>
      <p>
        After qualification many physiotherapists go on to train in specialist areas including acupuncture, pilates and animal therapy. As we have several physiotherapists at the clinic, with a range of different specialisms, we may transfer you to a different physiotherapist within the clinic whose skills, we believe, will more successfully treat your condition.
      </p>
      <h2>
        Common problems treated by Physiotherapists
      </h2>
      <ul>
        <li>
          <h3>
            Tennis Elbow
          </h3>
          <p>
            A common term used to describe pains in the forearm and elbow area. However this condition is not restricted to tennis players but can affect golfers, gardeners and factory workers who strain their arm. The condition can be treated with manual therapy, ultrasound and often the use of an arm brace can help prevent the condition returning.
          </p>
        </li>
        <li>
          <h3>
            Upper and lower back pain
          </h3>
          <p>
            These can have a variety of causes from poor posture to whiplash and even uneven leg length. Most can be successfully treated with a combination of manual therapy, advice, postural correction and exercises. Lumbar rolls, seat wedges and neck pillows can also help maintain improvement as can workplace assessments.
          </p>
        </li>
        <li>
          <h3>
            Headaches
          </h3>
          <p>
            Headaches are often caused by problems in the top part of the neck at the base of the skull. Headaches with this cause can therefore be treated as a neck problem and do respond well to manual therapy and exercises.
          </p>
        </li>
        <li>
          <h3>
            Strains
          </h3>
          <p>
            Muscle and ligament strains are very common and not just in sports men and women. They can be extremely painful and debilitating but fortunately do respond very well to electrotherapy such as ultrasound or laser treatment. Graded exercises and advice on how quickly to return to your normal activity are also very important.
          </p>
        </li>
        <li>
          <h3>
            Arthritis
          </h3>
          <p>
            Although the actual arthritis cannot be treated by physiotherapy, the pain, swelling and muscle weakness can. Physiotherapy can therefore make life much more comfortable and even delay or prevent the need for surgery.
          </p>
        </li>
        <li>
          <h3>
            Sports injuries
          </h3>
          <p>
            Often injuries occur during training and whilst rest would be the first choice treatment upcoming events e.g. a marathon or a football tournament don't allow this. Physiotherapy can speed up the healing process and specific training advice will ensure training can continue without causing further injury. RICE is the recognised first intervention, meaning rest, ice, compression and elevation. Physiotherapists are experts in the initial treatment and follow up rehabilitation of these types of injuries.
          </p>
        </li>
      </ul>
      <h2>
        Homeopathy
      </h2>
      <p>
        Homeopathy is a safe, complementary medicine suitable for all ages. It can be used to treat stress and anxiety, migraines, eczema, childhood illness and behaviour problems.
      </p>
      <h2>
        Steroid Injection Therapy
      </h2>
      <h3>
        Why would I need a local steroid injection?
      </h3>
      <p>
        Steroid Injections can be given to treat inflammation or pain. They are usually injected into the joint of the surrounding soft tissue. Sometimes they are used alone, other times they will be followed by physiotherapy. Reducing pain and swelling with a steroid injection can help make the physiotherapy work better. The steroid injection is usually combined with local anaesthetic. Local anaesthetic is used to increase the volume of the injection and help us know that the injection is in the right place.
      </p>
      <h3>
        How long will the steroid injection take to work, and how long will it last?
      </h3>
      <p>
        This varies between different people, but usually improvement starts after one or two days. Sometimes it can take up to 6 weeks for the benefits to take effect. The benefit usually lasts from a few weeks to several months.
      </p>
      <h3>
        Do I need to do anything after the injection?
      </h3>
      <p>
        After the injection it is advisable to rest the injected limb for the first 3 to 7 days. Try to avoid activities that make the symptoms worse. Limit walking and don't participate in sports.
      </p>
      <h2>
        Orthotics
      </h2>
      <p>
        Excessive pronation of the foot (flat feet) causes the arch of the foot to flatten and can result in heel, knee, hip and back pain. It is very common and in most cases can be treated by a physiotherapist with supportive insoles. The insoles support the arch, realigning the lower limb correctly which alleviates the resulting pain. Incorrect foot posture and leg length discrepancies can also be corrected with appropriately positioned additional wedges. For more complicated conditions we can refer you to a specialist podiatrist.
      </p>
      <p>
        The sequence of pictures below show: 1. Normal foot; 2. Flat foot with collapsed arch; 3. Over arched foot; 4. Patient with excess pronation commonly known as flat feet.
      </p>
    </main>
    <footer>
      <div>
        <div>
          <p>
            Find us on Social Media
          </p>
          <div>
            <a
              href="https://www.facebook.com/HighworthPhysio/"
              rel="noopener noreferrer"
              target="_blank"
            >
              <svg
                fill="none"
                height="32"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="32"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M18 2h-3a5 5 0 0 0-5 5v3H7v4h3v8h4v-8h3l1-4h-4V7a1 1 0 0 1 1-1h3z"
                />
              </svg>
            </a>
          </div>
        </div>
        <div>
          <p>
            All of our physios are registered with at least one of the following regulatory bodies:
          </p>
          <ul>
            <li>
              <a
                href="https://www.csp.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Chartered Society of Physiotherapists"
                >
                  CSP
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.hcpc-uk.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Health and Care Professions Council"
                >
                  HCPC
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.physiofirst.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                Physio First
              </a>
            </li>
            <li>
              <a
                href="https://www.aacp.org.uk/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Acupuncture Association of Chartered Physiotherapists"
                >
                  AACP
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.acpat.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Association of Chartered Physiotherapists in Animal Therapy"
                >
                  ACPAT
                </abbr>
              </a>
            </li>
            <li>
              <a
                href="https://www.rampregister.org/"
                rel="noopener noreferrer"
                target="_blank"
              >
                <abbr
                  title="Register of Animal Musculoskeletal Practitioners"
                >
                  RAMP
                </abbr>
              </a>
            </li>
          </ul>
        </div>
      </div>
    </footer>
  </div>,
}
`;
